src/commands/appSettings/uploadAppSettings.ts

```diff
@@ -50,10 +50,25 @@ export async function uploadAppSettingsInternal(context: IActionContext, client:
             remoteSettings.properties = {};
         }
 
+        const excludedAppSettings: string[] = [];
+        if (exclude) {
+            Object.keys(localSettings.Values).forEach((settingName) => {
+                if (exclude.some((exclusion) => typeof exclusion === 'string' ? settingName === exclusion : settingName.match(exclusion))) {
```

Comment:
Settings are case-insensitive so you should probably change the settingName to all lower case along with the exclusion when you do the comparisons.

Reply:
Good catch, changed it to `settingName.match(new RegExp(exclusion, 'i'))`. This creates a new RegExp that is case-insensitive.